README.md

```diff
@@ -112,7 +112,7 @@ to all Giant Swarm installations.
 
 - v13
   - v13.0
-    - [v13.0.0](https://github.com/giantswarm/releases/tree/master/azure/v13.0.0)
+    - [v13.0.0](https://github.com/giantswarm/releases/tree/master/azure/archived/v13.0.0)
```

Comment:
We archived the 13.0.0 release before we even announced it to customers?
Doesn't feel very smart